### ISSUE TITLE
Fetch All Update Comments

### ISSUE THREAD

**Issue by GoogleCodeExporter** [NityaNandPandey/Linkedin-SignIn#22]
```
The library currently fetches most recent 5 comments for each network
update. The LinkedIn API also includes call to fetch all comments. Need to
incorporate that in the library. Also the id and timestamp of an update
comment are not available through the library but they are available
through the API. This needs to be fixed as well.
```

Original issue reported on code.google.com by `nabeelmukhtar` on 2 Jan 2010 at 12:40

**Comment by GoogleCodeExporter** [comment]
Original comment by `nabeelmukhtar` on 2 Jan 2010 at 3:38
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This issue has been fixed in trunk and linkedin-j-0.1-SNAPSHOT-2010-01-04.zip.
```

Original comment by `nabeelmukhtar` on 4 Jan 2010 at 12:07
- Changed state: **Fixed**
